extensions/amp-analytics/0.1/vendors/deepbi.js

```diff
@@ -17,14 +17,14 @@
 import {jsonLiteral} from '../../../../src/json';
 
 const DEEPBI_CONFIG = jsonLiteral({
-  'requestOrigin':'https://${endpoint}/v1/pixel',
+  'requestOrigin':'https://${endpoint}',
```

Comment:
add space on right side of semi-colon (currently causing lint test to fail)

Reply:
Thank you! Pushed changes.